pootle/static/js/editor/app.js

```diff
@@ -915,6 +915,13 @@ PTL.editor = {
       this.checkSimilarTranslations();
       this.similarityTimer = null;  // So we know the code was run
     }, 200);
+
+    this.updateCharacterCount()
+  },
+
+  updateCharacterCount() {
+    let count = ReactEditor.stateValues.map(e => e.length).join(' \ ')
+    $('#char-count').html(count)
```

Comment:
Travis detects several errors with this Javascript code:

- Must use semicolons at end of relevant lines
- Use `const` instead of `let` because it never gets reassigned